jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -288,6 +293,18 @@ private static DiagnosticProperties runGC() {
 		return DiagnosticProperties.makeCommandSucceeded();
 	}
 
+	private static DiagnosticProperties getJstatClass(String diagnosticCommand) {
+		IPC.logMessage("jstat command : ", diagnosticCommand); //$NON-NLS-1$
+		StringBuilder resultSB = new StringBuilder();
+		resultSB.append("Class Loaded    Class Unloaded\n"); //$NON-NLS-1$
+		// "Class Loaded".length = 12
+		resultSB.append(String.format("%12s", new Integer(ClassLoaderInfoBaseImpl.getLoadedClassCountImpl()))); //$NON-NLS-1$
+		resultSB.append("    "); //$NON-NLS-1$
+		// "Class Unloaded".length = 14
+		resultSB.append(String.format("%14s", new Long(ClassLoaderInfoBaseImpl.getUnloadedClassCountImpl()))); //$NON-NLS-1$
+		return DiagnosticProperties.makeStringResult(resultSB.toString());
```

Comment:
This should a `StringWriter` wrapped in a `PrintWriter` to properly handle new-lines (instead of using `\n`) and so `format` can be used directly on that writer object (`%12s` should be `%12d` to better match the parameter).
This should use `Integer.valueOf()` rather than `new Integer()` (same goes for `Long`).

Reply:
Fixed.